herbert_core/utilities/misc/serialize.m

```diff
@@ -3,6 +3,9 @@
 [use_mex,fm] = config_store.instance().get_value('hor_config',...
     'use_mex','force_mex_if_use_mex');
 
+
+persistent wc;
```

Comment:
be really nice if comments told us what wc is using, why this cycles on 50 iters etc. Lots more comments please

Reply:
I've touched this thing erroneously, as was following serialize to see how it works with hashes. This is Jacob's routine which I have not changed and his change (`wc`) was to satisfy user request to produce limited number of warning  if something going wrong but not critically wrong. 

Probably 50 should be in `hor_config` but this needs discussion and another ticket. I would add couple of rows of description but there should be number of places with code like that.